packages/synapse-react-client/src/components/OAuthClientManagement/CreateOAuthClient.integration.test.tsx

```diff
@@ -44,10 +43,14 @@ const defaultProps: CreateOAuthModalProps = {
   setIsShowingConfirmModal: jest.fn(),
 }
 
-jest.spyOn(SynapseClient, 'createOAuthClient')
-jest.spyOn(SynapseClient, 'updateOAuthClient')
-const createOAuthClientSpy = jest.mocked(SynapseClient.createOAuthClient)
-const updateOAuthClientSpy = jest.mocked(SynapseClient.updateOAuthClient)
+const createOAuthClientSpy = jest.spyOn(
+  MOCK_CONTEXT_VALUE.synapseClient.openIDConnectServicesClient,
+  'postAuthV1Oauth2Client',
+)
+const updateOAuthClientSpy = jest.spyOn(
+  MOCK_CONTEXT_VALUE.synapseClient.openIDConnectServicesClient,
+  'putAuthV1Oauth2ClientId',
+)
```

Comment:
In test, I want to verify that we sent the new data to the server, without testing the implementation of the mock service. We can still `spyOn` the method, but we need to get the specific instance of `synapseClient` used to make the call.